core/ontology/generis.rdf

```diff
@@ -381,4 +381,30 @@
     <rdfs:domain rdf:resource="http://www.w3.org/1999/02/22-rdf-syntax-ns#Property"/>
     <rdfs:range rdf:resource="http://www.w3.org/2000/01/rdf-schema#Property"/>
   </rdf:Description>
+
+  <rdf:Description rdf:about="http://www.tao.lu/Ontologies/generis.rdf#InterfaceModes">
+      <rdfs:subClassOf rdf:resource="http://www.tao.lu/Ontologies/TAO.rdf#SystemObject"/>
```

Comment:
We need to think about using a different resource as parent, because this one relates to `TAO.rdf`, which belongs to child repository (generis -> tao).

Reply:
@Nevermind23 we probably need to use `http://www.tao.lu/datatypes/WidgetDefinitions.rdf#RadioBox`